### DISCUSSION TITLE
can not extract keycloak-docker-compose-yaml.zip exported by docker client

### DISCUSSION THREAD

**Discussion by Ali-Javanmardi** [keycloak/keycloak#13723] [Q&A]
Hi 
I tried to connect a docker registry to keycloak quarkus 19.0.1 by the documentation: https://www.keycloak.org/docs/latest/securing_apps/#configuring-a-docker-registry-to-use-keycloak

In keycloak admin panel I have no problem and can export and download the file keycloak-docker-compose-yaml.zip

However when I try to extract that file by all archive managers in my linux machine I receive archive file error.
I even tried 7zip by wine and it has same error.
The file size I received from keycloak is about 8.2KB and as I have minimum initial config in that client seems reasonable.
Is there any problem in compression method in that export procedure or I should use any special zip format?

**Comment by 0xpr03** [comment]
Could be this be the same issue as mine ? https://github.com/keycloak/keycloak/discussions/13683

**Reply by Ali-Javanmardi** [reply]
It seems to be same.
If there is a problem with compression library it may affect any compressed data.

**Reply by 0xpr03** [reply]
I filed https://github.com/keycloak/keycloak/issues/13733
